docker/build

```diff
@@ -2,17 +2,7 @@
 
 set -e
 
-if [[ $BUILD == dub ]]; then
-    dub build --build=release
-
-    mkdir bin
-    mv dfmt ./bin
-elif [[ $DC == ldc2 ]]; then
-    git submodule update --init --recursive
-    make ldc -j2
-else
-    git submodule update --init --recursive
-    make pkg -j2
-fi
+git submodule update --init --recursive
+make pkg -j2
```

Comment:
This doesn't belong to `docker/build`, again this is just to build the docker image, not the project!